support/pal/Cargo.toml

```diff
@@ -43,14 +44,20 @@ features = [
     "securitybaseapi",
     "std",
     "synchapi",
-    "threadpoolapiset",
     "winbase",
     "winerror",
     "winioctl",
     "winnt",
     "winsock2",
 ]
 
+[target.'cfg(windows)'.dependencies.windows]
```

Comment:
Usually for low-level stuff I prefer `windows_sys` so that we don't have to fight whatever wrappers `windows` adds.